hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/dto/AttributeSearchRequestJsonTest.kt

```diff
@@ -296,14 +307,12 @@ class AttributeSearchRequestJsonTest(@Autowired val objectMapper: ObjectMapper)
         matchers = listOf(
           Matchers(
             joinType = JoinType.AND,
-            stringMatchers = listOf(
+            matchers = listOf(
               StringMatcher(
                 attribute = "firstName",
                 condition = TextCondition.IS,
                 searchTerm = "John",
               ),
-            ),
-            booleanMatchers = listOf(
```

Comment:
With a single list of matchers we don't discrimate by matcher type which felt a bit clunky